views/auth/login.hbs

```diff
@@ -0,0 +1,23 @@
+<div class="row">
+    <div class="col-12">
+        <form action="/login" method="POST">
+          <div class="login-div">
+            <label for="username">Username:</label>
+            <input type="text" name="username">
+          </div>
+
+          <div class="login-div">
+            <label for="password">Password:</label>
+            <input type="password" name="password">
+          </div>
+
+          {{#if errorMessage}}
+          <div class="error-message">{{errorMessage}}</div>
+          {{/if}}
+
+          <div class="login-div">
+            <a id="btn-a" href="/user-profile">Login</a>
+          </div>
```

Comment:
The ```a``` link here will always route the user to "/user-profile" instead of submitting the form. Ideally you want to use a ```button``` tag here so that the form is submitted.

```suggestion
          <div class="login-div">
            <button type="submit">Login</button>
          </div>
```